src/database.ts

```diff
@@ -3448,9 +3476,13 @@ class Database extends common.GrpcServiceObject {
 
     let sessionId = '';
     const getSession = this.pool_.getSession.bind(this.pool_);
+
     return startTrace(
       'Database.runTransactionAsync',
-      this._traceConfig,
+      {
+        ...this._traceConfig,
+        transactionTag: options?.requestOptions?.transactionTag,
```

Comment:
should we add a test for this in observability-test/database.ts ?

Reply:
done